tango/sweeps.py

```diff
@@ -0,0 +1,68 @@
+import itertools
+import json
+import subprocess
+from collections import OrderedDict
+from dataclasses import dataclass
+
+from tango.common import Params, Registrable
+
+
+class Sweeper(Registrable):
+    def __init__(
+        self,
+        main_config_path: str,
+        sweeps_config_path: str,
+        components: str,
+    ):
+        super(Registrable, self).__init__()
```

Comment:
In recent versions of Python you can just do this:
```suggestion
        super().__init__()
```